consoleme/handlers/v2/audit.py

```diff
@@ -28,6 +35,33 @@ async def get(self):
         """
         GET /api/v2/audit/roles
         """
+        log_data = {
```

Comment:
Should we have authz on this endpoint on which app identities can access?

Reply:
I did this with the initial implementation, but there was complexity around app vs user identity and it would have been a pain to manage the access. It turned out to not be worth it since any authed user has access to this data via the policies table anyway.